### PR TITLE
Consolidate and update organizational documents

### PR DESCRIPTION
**Pull request by mwaskom**
This PR does housekeeping on the organizational documents in the repo. It removes some outdated/redundant documents and consolidates information about the tutorial production pipeline into a single prose document and the style guide noteboook.

### CODE REVIEW

File: organization/Tutorial_Pipeline.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Content Review
+
+After tutorials have been created, they are evaluated by several Reviewers. The review process focuses on the accuracy and pedagogical effectiveness of the content, including videos, prose, math, and code. Ideally, Reviewers will have domain-specific expertise relating to the days that they volunteer to review, although non-experts can provide helpful feedback about clarity and organization. Reviewing and Editing are partially overlapping, but Reviewers should focus on content, not style.
+
+Content Review is organized externally to the GitHub repository using a Google Sheet that tracks the URLs for the latest version of each notebook, the associated tutorial videos, and the identities of the various parties involved. There is also a slack channel (`#content-reviews`) where review assignments are coordinated.
```
Comment (patrickmineault) [inline organization/Tutorial_Pipeline.md:27]: Next year we'll use an airtable instead of a Google sheet.

**Review by patrickmineault — APPROVED** [review]
These instructions look great. I would like to see a second opinion from @ebatty since she was more deeply involved.

**Review by ebatty — COMMENTED** [review]
This looks great to me! The only thing potentially missing from the tutorial publication description is the conversion and publication for China - should we include that here?

**Comment by mwaskom** [comment]
Hm my sense is that that should be documented separately but I don't feel strongly.

**Comment by ebatty** [comment]
Yeah that's fine with me - merge away!

**Pull request merged**